src/main/java/uk/gov/justice/hmpps/prison/service/AdjudicationService.java

```diff
@@ -182,18 +181,6 @@ private int getAdjudicationCount(LocalDate awardCutoffDateParam, LocalDate adjud
         return adjudicationCount;
     }
 
-    public List<ProvenAdjudicationSummary> getProvenAdjudications(final List<Long> bookingIds,
-                                                                  final LocalDate cutoffDateParam) {
-        return repository.findAwardsForMultipleBookings(bookingIds).stream()
```

Comment:
repository.findAwardsForMultipleBookings can be removed

Reply:
FIND_AWARDS_BY_BOOKINGS enum can be removed too from `AdjudicationRepositorySql.kt`

Reply:
good spot - addressed with https://github.com/ministryofjustice/prison-api/pull/1720/commits/003c1df263c73ffaf74f4870f0eb9f9418ce4508